python/docs/conf.py

```diff
@@ -0,0 +1,203 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+
+# -- Project information -----------------------------------------------------
+
+project = "rmm"
+copyright = "2020, NVIDIA"
+author = "NVIDIA"
+
+# The full version, including alpha/beta/rc tags
+release = "0.17"
+
+
+# -- General configuration ---------------------------------------------------
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+
+extensions = [
+    "sphinx.ext.intersphinx",
+    "sphinx.ext.autodoc",
+    "sphinx.ext.autosummary",
+    "sphinx_copybutton",
+    "numpydoc",
+    "sphinx_markdown_tables",
+    "IPython.sphinxext.ipython_console_highlighting",
+    "IPython.sphinxext.ipython_directive",
+    "nbsphinx",
+    "recommonmark",
+    "sphinx.ext.napoleon",
+]
+
+
+copybutton_prompt_text = ">>> "
+
+ipython_mplbackend = "str"
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ["_templates"]
+
+# The suffix(es) of source filenames.
+# You can specify multiple suffix as a list of string:
+#
+# source_suffix = ['.rst', '.md']
+source_suffix = {".rst": "restructuredtext", ".md": "markdown"}
+
+# The master toctree document.
+master_doc = "index"
+
+# General information about the project.
+project = "rmm"
+copyright = "2020, NVIDIA"
+author = "NVIDIA"
+
+# The version info for the project you're documenting, acts as replacement for
+# |version| and |release|, also used in various other places throughout the
+# built documents.
+#
+# The short X.Y version.
+version = "0.17"
+# The full version, including alpha/beta/rc tags.
+release = "0.15.0"
```

Comment:
This looks wrong?

Reply:
Would also be good to capture these in https://github.com/rapidsai/rmm/blob/branch-0.17/ci/release/update-version.sh

Reply:
Fixed & done